app/lib/dependent_relationship_qualifier.rb

```diff
@@ -0,0 +1,19 @@
+class DependentRelationshipQualifier
```

Comment:
let's call this something like "ChildRelationshipQualifier", since the other relationships can still be valid dependents but they aren't "children"